src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,31 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class HomePage {
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226275894-6b533c93-817f-442c-8fee-d2b2545f54d1.png)

HINT

1. public class HomePage **extends BasePage** 
2. WebElement playlistNameTextField = driver.findElement(playlistInputField);
3. public void doubleClickFirstPlaylist(String playlistName)